src/components/WalletsPage/index.js

```diff
@@ -0,0 +1,111 @@
+        <Hidden xsDown implementation="css">
+          <div style={{display: activeWallet ? 'none' : 'block'}}>
+            <Typography variant="h5" style={{padding: 40}}>Please select a wallet</Typography>
+          </div>
+        </Hidden>
+        <div style={{width: '100%', display: activeWallet ? 'block' : 'none'}}>
+          <Hidden xsDown implementation="css">
+            <Typography
+              variant="h4"
+              classes={{ h4: classes.currencyName }}
+              gutterBottom
+            >
+              {activeWalletName}
+              <Typography
+                variant="alignRight"
+                classes={{ alignRight: classes.totalBalance }}
+                gutterBottom
+              >Total Balance: <b>{toMinFixed(activeWalletData.balance, 2)}</b></Typography>
+              <Typography
+                variant="alignRight"
+                classes={{ alignRight: classes.locked }}
+                gutterBottom
+              >Locked: {toMinFixed(activeWalletData.locked, 2)}</Typography>
+            </Typography>
+            <Divider />
+            <Tabs
+              value={Math.max(['/wallets/deposit', '/wallets/withdrawal'].indexOf(location.pathname), 0)}
+              indicatorColor="primary"
+              textColor="primary"
+            >
+              <Tab
+                label="Deposit"
+                classes={tabClasses}
+                component={Link}
+                to={{pathname: '/wallets/deposit', search: location.search}}
+                icon={<VerticalAlignTopIcon className={classes.verticalMiddle} />}
+              />
+              <Tab
+                label="Withdrawal"
+                classes={tabClasses}
+                component={Link}
+                to={{pathname: '/wallets/withdrawal', search: location.search}}
+                icon={<VerticalAlignBottomIcon className={classes.verticalMiddle} />}
+              />
+            </Tabs>
+          </Hidden>
+          <Hidden smUp implementation="css">
+            <Tabs
+              value={Math.max(['/wallets/deposit', '/wallets/withdrawal'].indexOf(location.pathname), 0)}
+              indicatorColor="primary"
+              textColor="primary"
+              classes={{root: classes.mobileTabs}}
+              fullWidth
+            >
+              <Tab
+                label="Deposit"
+                component={Link}
+                to={{pathname: '/wallets/deposit', search: location.search}}
+              />
+              <Tab
+                label="Withdrawal"
+                component={Link}
+                to={{pathname: '/wallets/withdrawal', search: location.search}}
+              />
+            </Tabs>
+          </Hidden>
+          <Divider />
+          <div className={classes.inner}>
+            <Switch>
+              <Route path={'/wallets/deposit'} exact component={Deposit} />
+              <Route path={'/wallets/withdrawal'} component={Withdraw} />
```

Comment:
@AntonBurduzha do you think route can be in the container WalletPage ?

Reply:
sure, we can pass <Switch> to Wallet component as `props.children`